### PR TITLE
Update mcr.cpp

### PR DESCRIPTION
**Pull request by Chen30lv**

### CODE REVIEW

**Comment by wychen-null** [comment]
you did well

File: mcr.cpp [hunk 1]
```diff
@@ -15,19 +19,35 @@ bool isWin(char game[3][3]){
 
 int main(){
     int i, j;
-    char game[3][3] = {' '}; // Tic-tac-toe
-    char player1 = 'X';
-    char player2 = 'O';
+    i = -1;
```
Comment (guestzz) [inline mcr.cpp]: I think this modification is unecessary. Line 23 is the same.

**Comment by Chen30lv** [comment]
> you did well

thank you!

File: mcr.cpp [hunk 2]
```diff
@@ -1,3 +1,7 @@
+#include "iostream"
+
```
Comment (wychen-null) [inline mcr.cpp:2]: you did well! GOOD!!!!

File: mcr.cpp (hunk 1 above)
Comment (Chen30lv) [reply mcr.cpp]: ok, I will update the code

**Review by wychen-null — COMMENTED** [review]
yes

File: mcr.cpp (hunk 2 above)
Comment (Chen30lv) [reply mcr.cpp:2]: hhh, than you

File: mcr.cpp [hunk 3]
```diff
@@ -15,19 +19,35 @@ bool isWin(char game[3][3]){
...
+        else{
             cout << "Player 2: ";
-        cout << "Which cell to mark? i:[1..3], j:[1..3]: ";
+        }
+
+        rewrite:
+        cout << "Which cell to mark? i:[0..2], j:[0..2]: ";
         cin >> i >> j;
+
+        if(game[i][j] != 'X' && game[i][j] != 'O' && i <= 2 && i >= 0 && j <= 2 && j >= 0) { //check if the input is appropriate.
```
Comment (KOBEliever) [inline mcr.cpp:39]: Still need to determine if the input is not a number and output a hint

File: mcr.cpp (hunk 2 above)
Comment (guestzz) [reply mcr.cpp:2]: 🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮🐮

File: mcr.cpp (hunk 3 above)
Comment (Chen30lv) [reply mcr.cpp:39]: I really have no idea with this, we just hope that no one will use characters.

**Pull request merged**